src/installer/installer.go

```diff
@@ -307,19 +307,28 @@ func (i *installer) waitForControlPlane(ctx context.Context, kc k8s_client.K8SCl
 		i.log.Error("failed to get network type")
 		return err
 	}
+	var origControlPlaneReplicas int
 	if nt == ovnKubernetes {
-		if err = kc.PatchControlPlaneReplicas(); err != nil {
-			i.log.WithError(err).Error("Failed to patch control plane replicas")
+		if origControlPlaneReplicas, err = kc.GetControlPlaneReplicas(); err != nil {
+			i.log.WithError(err).Error("Failed to get control plane replicas")
 			return err
 		}
+		if origControlPlaneReplicas == 3 {
```

Comment:
Use a const for 3.
Also, add a comment on why it's required.

Reply:
OK